instrumentation/tomcat/tomcat-10.0/javaagent/src/test/groovy/io/opentelemetry/javaagent/instrumentation/tomcat/v10_0/TomcatHandlerTest.groovy

```diff
@@ -75,6 +76,20 @@ class TomcatHandlerTest extends HttpServerTest<Tomcat> implements AgentTestTrait
 
   @Override
   void responseSpan(TraceAssert trace, int index, Object parent, String method, ServerEndpoint endpoint) {
+    SpanData span = trace.span(0)
+    String requestHeaders = "";
+    String responseHeaders = "";
+    for (Map.Entry<AttributeKey, Object> entry : span.getAttributes().asMap().entrySet()) {
+      if (entry.key.key == "http.request.headers") {
+        requestHeaders = entry.value;
+      } else if (entry.key.key == "http.response.headers") {
+        responseHeaders = entry.value;
+      }
+    }
+
+    assert requestHeaders.length() > 0
```

Comment:
where do you add headers? are we sure there are any?

Reply:
In case there aren't any, we'll have an empty dict string.